kibble/models/site_brand.go

```diff
@@ -0,0 +1,38 @@
+//    Copyright 2018 SHIFT72
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package models
+
+type SiteBrand struct {
+	Images map[string]string
+	Links  map[string]string
+}
+
+//Get URL for SiteBrand item, return a passed in default path if non-existant
+func (s *SiteBrand) GetImage(assetType string, defaultPath string) string {
+
+	return getItem(s.Images, assetType, defaultPath)
+}
+
+func (s *SiteBrand) GetLink(assetType string, defaultPath string) string {
+
+	return getItem(s.Links, assetType, defaultPath)
+}
+
+func getItem(brandType map[string]string, assetType string, defaultPath string) string {
```

Comment:
Given that you have called the key `assetType`, I think `assetMap` (or something similar) would be a more suitable name for `brandType` here.

Reply:
I also think that `defaultPath` should be renamed to `defaultURL` in this file.